server/protocols/jmap-draft/src/main/java/org/apache/james/jmap/draft/methods/MessageSender.java

```diff
@@ -60,7 +60,7 @@ public void sendMessage(MessageFactory.MetaDataWithContent message,
     }
 
     @VisibleForTesting
-    static Mail buildMail(MessageFactory.MetaDataWithContent message, Envelope envelope) throws MessagingException {
+    static Mail buildMail(MessageFullViewFactory.MetaDataWithContent message, Envelope envelope) throws MessagingException {
```

Comment:
Can we import MessageFullViewFactory.MetaDataWithContent so that we don't have the MessageFullViewFactory prefix?